images/router/haproxy/run_haproxy

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# make a pem file from the tls.* files
+echo $DEFAULT_CERTIFICATE_DIR
+if [ -f $DEFAULT_CERTIFICATE_DIR/tls.crt ]
```

Comment:
@liggitt The needed pem construction will be done in the router.go so this file will be deleted.